Dev

VladislavMaksimov opened the pull request:
Сделал боковое меню за исключением того значка у выбранного элемента и количества пропущенных задач (делаются скриптом) и fixed позиционирования.

VladislavMaksimov committed:
Merge pull request #1 from ENAleksey/master

Create README.md

Commit:
Исправил названия некоторых переменных. Переместил font-size из :root в html.

Commit:
Добавил цвет фона.

Commit:
Поменял фон бокового меню, добавил ему скругление углов и убрал границы.

Commit:
Добавил ширину бокового меню (без отступа нельзя использовать). Привёл селекторы к виду, прописанному в правилах вёрстки.

Commit:
Добавил цвет фона выбранному элементу бокового меню.

Commit:
Центрировал текст в боковом меню, задал высоту его элементам. Расставил табы перед !important, чтобы код лучше читался. Сделал селекторы более специфичными, чтобы не стайлить лишние элементы.

Commit:
Удалил отрицательный отступ у элементов бокового меню. Поменял стиль разноцветных элементов бокового меню, кнопки количества опросов/сообщений от преподов/etc., шрифт у выбранного элемента, изменение цвета его фона при наведении мыши.

Commit:
Скрыл хэдер. Добавил отступ для контента справа.Скрыл хэдер. Добавил отступ для контента справа. Добавил простейший медиазапрос.

VladislavMaksimov commented:
![image](https://user-images.githubusercontent.com/46715602/100622103-472e3200-3342-11eb-8687-eef46959c049.png)
Не забыть поправить правый паддинг.

MatushkinaAnastasia committed:
Сделан раздел "Пропущенные занятия" и начат раздел "Объявления"

Добавлены изменения в .css и .js. Всё вместе запускать.

MatushkinaAnastasia committed:
Добавлена обработка элементов раздела "Пропущенные занятия"

Закомментировала первый блок кода, так как без пикчи ничего работать не будет. 
Код в скрипте меняет цвет текста "Всего пропущенных занятий: n" и скругляет края таблицы.

VladislavMaksimov commented:
@MatushkinaAnastasia со скруглением углов есть проблема:
![image](https://user-images.githubusercontent.com/46715602/100631712-d4c34f00-334d-11eb-9862-f5575fddd3cc.png)

Вы подменяете скриптом и у верхней, и у нижней строк все 4 угла. Из-за этого, когда строка одна, возникает такой баг.

Ну и в целом для скругления скрипты совсем необязательны, потому что есть псевдоклассы nth-child и nth-last-child:
https://developer.mozilla.org/ru/docs/Web/CSS/:nth-child
https://developer.mozilla.org/ru/docs/Web/CSS/:nth-last-child

VladislavMaksimov commented:
![image](https://user-images.githubusercontent.com/46715602/100632105-400d2100-334e-11eb-8e41-38b189047df7.png)

И, кажется, мы эти вкладки хотели сделать с пунктирным почёркиванием, как изначально в етисе было. Просто в фигме нашли только одно подчёркивание.

@ENAleksey @MatushkinaAnastasia

VladislavMaksimov commented:
![image](https://user-images.githubusercontent.com/46715602/100632363-8cf0f780-334e-11eb-996e-51ca6c137636.png)

Кстати, вот это ведь можно скриптом сделать? Просто подсчитав красные и зелёные даты, которые етис проставляет сам. Ну и выводить только если есть по уважительной причине пропуски.

ENAleksey commented on styles.css:
```diff
@@ -1,7 +1,8 @@
+html {
+	font-size: 10px		!important;
```
Почему в :root нет переменной для размера шрифта?

VladislavMaksimov replied:
А зачем она нужна?

ENAleksey commented on styles.css:
```diff
@@ -13,25 +14,191 @@
+
+@media screen and (max-width: 575px) {
+	/**** БОКОВОЕ МЕНЮ ****/
+	div.span3 > .nav.nav-tabs {
+		display: none	!important;
+	}
```
Почему скрывается div.span3 > .nav.nav-tabs, а не span3?

VladislavMaksimov replied:
Учёл.

ENAleksey commented on styles.css:
```diff
@@ -13,25 +14,191 @@
+}
+
+/* Индикатор количества опросов/сообщений/etc. у бокового меню */
+span.badge {
```
.badge используется не только в боковом меню, нужно указывать селектор .span3 .badge

VladislavMaksimov replied:
Учёл.

ENAleksey commented on styles.css:
```diff
@@ -13,25 +14,191 @@
+
+	/*******ПРОПУЩЕННЫЕ ЗАНЯТИЯ*********/	
+
+div.span9 {
```
Зачем везде дополнительно указывать названия элементов, а не просто названия классов?

VladislavMaksimov replied:
Это ещё в правилах вёрстки было записано. Просто для того, чтобы с меньшей вероятностью стайлить лишнее.

ENAleksey commented:
Исправить padding у многострочного текста в боковом меню, он должен быть, как и у остальных элементов.

![image](https://user-images.githubusercontent.com/12061580/100716759-f3bbf280-33da-11eb-8476-e1b12f465a52.png)

ENAleksey commented:
Отличается цвет у ссылок бокового меню при состоянии hover.

![image](https://user-images.githubusercontent.com/12061580/100716855-14844800-33db-11eb-88d1-8b51141e1ec5.png)
![image](https://user-images.githubusercontent.com/12061580/100716867-177f3880-33db-11eb-80e7-38bde0958540.png)

ENAleksey commented:
Я считаю, что нужно всё по максимуму делать через css, все эти изменения можно сделать и без js. Если делать через скрипты, то внедрение стилей может осуществляться с задержкой, в итоге мы будем видеть как стили подменяются на наших глазах.

MatushkinaAnastasia committed:
Раздел "Объявления", "Учебный план" (не до конца сделан). Доработан раздел "Пропущенные занятия".

Многие таблицы поменяли вид и в целом ЕТИС стал выглядеть лучше.
Читайте комментарии, пишите фидбэки.

MatushkinaAnastasia committed:
Удалены строки, меняющие углы. Добавлен костыль, делающий отсупы между абзацами (Раздел "Объявления", "Сообщения от преподаваталей").

Кто знает как сделать отступы лучше - напишите.

Commit:
Доделал скругление углов с помощью стилей. Углы скругляются вне зависимости от количества строк в таблице.

Commit:
Установил правый паддинг для элементов бокового меню. В медиазапросе скрываю не .nav.nav-tabs, а .span3. Исправил селектор бейджа в боковом меню. Исправил цвет ховера для "Пожалуйста, оцените..."

VladislavMaksimov commented:
@ENAleksey запушил изменения. Там я исправил то, у чего написал "учёл", плюс цвет ховера "Пожалуйста, оцените..."

И добавил правый паддинг для элементов, чтобы на бейдж не залезали. Но если в элементе с бейджем будет нежирный текст "Сообщения от преподавателей", он всё равно будет заканчиваться очень близко к бейджу. Можешь чекнуть через инструменты разраба.

Про padding у многострочного текста не понял: что нужно исправить?

ENAleksey commented:
@VladislavMaksimov, у бокового меню высота ссылки не адаптируется под текст
![image](https://user-images.githubusercontent.com/12061580/101244787-327ae100-372a-11eb-8352-b23e36c8127d.png)

VladislavMaksimov commented:
Контент в span9 находится ближе к span3:

- Моё расписание
- Приказы
- Преподаватели
- Обратная связь (скорее всего)
- Журнал посещений
- Объявления
- Сообщения от преподавателей
- Образовательный стандарт
- Заказ справок
- Портфолио
- О ресурсе
- Анкетирование по прошедшему триместру
- Опросы
- Пожалуйста, оцените...
- Выбор факультативов
- Смена пароля

Контент span9 дальше от span3:

- Учебный план
- Мои оценки
- Пропущенные занятия
- Библиотека
- Электронные ресурсы
- Смена личной записи

VladislavMaksimov commented on styles.css:
```diff
@@ -111,12 +143,56 @@ div.span9 > .msg {
+.common th {
+    color: var(--color-text-primary) !important; 
+	background-color: var(--color-card) !important;
+	border: 0.24rem solid var(--color-body) 	!important;
```
Похоже, проблема в таких строках. Границу надо не переводить в очень маленький размер (0.24rem = 2.4px для нас, как посчитается 0.4px - хз) и присваивать им цвет фона, а убирать: `border: 0px !important;`

warmbl committed:
Update styles.css

Сделала страницы Приказы, Оценки за сессии. Сделала один цвет для всех ссылок в ЕТИСе.

ENAleksey commented:
Страница с оценками
=
### 1. Таблица не растягивается на всю ширину.
![image](https://user-images.githubusercontent.com/12061580/101260941-307a4780-3755-11eb-93a1-7c15d4daba6b.png)
### 2. У даты не изменён цвет.
![image](https://user-images.githubusercontent.com/12061580/101260895-ef823300-3754-11eb-8ad5-dfa3e725f281.png)
### 3. Имена преподавателей не выровнены по центру.
![image](https://user-images.githubusercontent.com/12061580/101260901-f6a94100-3754-11eb-9f41-76beb16bde1c.png)
### 4. Неправильный padding у строк таблицы.
![image](https://user-images.githubusercontent.com/12061580/101260958-4982f880-3755-11eb-85cf-ec35ea9ff331.png)
![image](https://user-images.githubusercontent.com/12061580/101260832-6ec33700-3754-11eb-814a-e8dbc1d8585e.png)
### 5. Нет деления на отдельные оценки с триместрами.
В етисе всё находится в одной таблице. Возможно, так и оставим.

![image](https://user-images.githubusercontent.com/12061580/101260984-7cc58780-3755-11eb-8e2e-fa5f09ecaaae.png)
![image](https://user-images.githubusercontent.com/12061580/101260975-70412f00-3755-11eb-84c0-3a918c65e4c8.png)

ENAleksey commented:
Приказы
==
Слишком большие отступы между приказами.

![image](https://user-images.githubusercontent.com/12061580/101261111-97e4c700-3756-11eb-8b59-d4a7a1dbae9c.png)

warmbl committed:
Update styles.css

Отступы в приказах сделала 3rem (вместо 5rem).

ENAleksey commented on styles.css:
```diff
@@ -300,8 +300,8 @@ body > div.container > div.row > div.span9 {
 div.span9 > ul.orders > li.ord {
-	margin-bottom: 5rem;
-	margin-bottom: 5rem;
+	margin-bottom: 3rem;
+	margin-bottom: 3rem;
```
1. Зачем два раза прописывать одно и то же?
2. Отступы слишком большие. У нас есть макет, всё берём оттуда.

VladislavMaksimov replied:
Мб нам увеличить немного отступы на макете?

ENAleksey replied:
Ссылки не реагируют на наведение курсора мыши. Необходимо уберать подчёркивание.

warmbl commented:
@

> # Приказы
> Слишком большие отступы между приказами.
> 
> ![image](https://user-images.githubusercontent.com/12061580/101261111-97e4c700-3756-11eb-8b59-d4a7a1dbae9c.png)

Исправила.

ENAleksey commented:
Мой учебный план
==
1. Неправильный padding
2. У первого заголовка не изменён цвет
3. Все тексты, кроме названия дисциплины, должны быть выровнены по центру и горизонтально, и вертикально

![image](https://user-images.githubusercontent.com/12061580/101261338-6240dd80-3758-11eb-9bed-814bb0e3da07.png)

ENAleksey commented:
Пропущенные занятия
==
1. Заголовки должны быть выровнены по центру горизонтально и вертикально
2. Увеличить ширину столбца Дата занятия, чтобы не было переноса на новую строку

Commit:
Добавил паддинги для ссылок в боковом меню. Теперь отступы работают одинаково для однострочного и многострочного текста.

VladislavMaksimov commented:
> @VladislavMaksimov, у бокового меню высота ссылки не адаптируется под текст
> ![image](https://user-images.githubusercontent.com/12061580/101244787-327ae100-372a-11eb-8352-b23e36c8127d.png)

Исправил.

VladislavMaksimov commented:
Заметил очередную проблему: у элементов этих списков при нажатии на них не добавляется класс active.

![image](https://user-images.githubusercontent.com/46715602/101278215-d6709500-37db-11eb-9612-1ee3d26d0bf1.png)

ENAleksey commented:
> Заметил очередную проблему: у элементов этих списков при нажатии на них не добавляется класс active.
> 
> ![image](https://user-images.githubusercontent.com/46715602/101278215-d6709500-37db-11eb-9612-1ee3d26d0bf1.png)

Нужно будет вручную проставлять.

VladislavMaksimov commented:
@MatushkinaAnastasia при выполнении скрипта с insertBefore вылетает ошибка:
![image](https://user-images.githubusercontent.com/46715602/101278705-2d2b9e00-37df-11eb-93a2-312b4109789c.png)

ENAleksey commented:
@VladislavMaksimov
1. На активной вкладке бокового меню текст бейджа не должен быть жирным
2. Размер бейджа слишком маленький, не по макету

![image](https://user-images.githubusercontent.com/12061580/101278972-b394af80-37e0-11eb-9630-8b115bb2752a.png)

VladislavMaksimov commented:
> > Заметил очередную проблему: у элементов этих списков при нажатии на них не добавляется класс active.
> > ![image](https://user-images.githubusercontent.com/46715602/101278215-d6709500-37db-11eb-9612-1ee3d26d0bf1.png)
> 
> Нужно будет вручную проставлять.

Короче, это задача на потом. Там каждый раз перезагружается страница, и видимо придётся использовать sessionStorage, чтобы запоминать значение. Но элемент DOM-дерева запомнить в него нельзя.

Возможно, стоит в цикле, который ниже написан, увеличивать индекс i и запоминать его, а при перезагрузке страницы по селектору с :nth-child(i) находить нужный элемент и менять ему шрифт. Пока код ниже не работает. Эту задачу закину в миро в доработки по второму спринту.

```
let thisWrongLink;

window.addEventListener('load', () => {
	if (typeof thisWrongLink != 'undefined')
		thisWrongLink.style.fontWeight = 'bold';
})

const wrongLinks = document.querySelectorAll("div.span3 > ul:not(:first-child) > li > a");

wrongLinks.forEach(link => {
	link.addEventListener('click', () => {
		alert(link)
		thisWrongLink = link;
		alert(thisWrongLink)
	})
})
```

VladislavMaksimov commented:
Плашку с макета тоже закину в доработки.
![image](https://user-images.githubusercontent.com/46715602/101279518-9feb4800-37e4-11eb-949a-14dabd177529.png)

Commit:
Сделал так, чтобы шрифт бейджа у активного элемента не становился жирным. Увеличил размер бейджа в соответствии с макетом.

VladislavMaksimov commented:
> @VladislavMaksimov
> 
> 1. На активной вкладке бокового меню текст бейджа не должен быть жирным
> 2. Размер бейджа слишком маленький, не по макету
> 
> ![image](https://user-images.githubusercontent.com/12061580/101278972-b394af80-37e0-11eb-9630-8b115bb2752a.png)

Исправил. С боковым меню пока закончим, иначе я расписание не успею сделать.

Между делом попробую position: fixed ему добавить, разве что.

MatushkinaAnastasia commented:
> @MatushkinaAnastasia при выполнении скрипта с insertBefore вылетает ошибка:
> ![image](https://user-images.githubusercontent.com/46715602/101278705-2d2b9e00-37df-11eb-93a2-312b4109789c.png)

**Это ок**

MatushkinaAnastasia committed:
Доработка дизайна разделов: пропущенные занятия, учебный план, сообщения от преподавателей.

Сделано:

Пропущенные занятия:
- выравнивание по центру (вертикально/горизонтально)

Учебный план:
- скругления краев таблицы (кривовато)
- выравнивание по центру (вертикально/горизонтально)
- паддинг между тримами меньше, паддинг в тексте чуть побольше

Сообщения от преподавателей:
- убран отступ перед ФИО преподавателя
- начат дизайн кнопки (она вылетает из формы, хз что делать)

MatushkinaAnastasia committed:
Обновление

Изменение раздела сообщения от преподавателей. Оформлены номера страниц. Сделано предложение от Лёхи насчет цвета активных ссылок.

warmbl committed:
Логотип ПГНИУ

warmbl committed:
Добавила удаление текста на страницу входа.

warmbl committed:
Сделала страницу входа

ENAleksey commented:
Страница входа
==
1.  svg логотипа с прозрачными линиями, поэтому видно, что они друг на друга накладываются. Нужно сделать его полностью непрозрачным, прозрачность задавать через стили.
2. Курсор в полях ввода черный.
3. Неправильные отступы у текста в полях ввода.
4. Шрифт текста в полях ввода отличается от остального.
5. Ссылка для регистрации должна быть вертикально выровнена по середине кнопки.
6. Информационный текст должен быть прикреплён к низу страницы.
7. Форма входа должна всегда центрироваться по вертикали страницы с небольшим смещением вверх.
8. Нет одной строки с информацией.
9. Нет тени у формы входа.
10. Видны белые углы у полей ввода при автозаполнении.

ENAleksey commented:
@MatushkinaAnastasia 

**Кнопки выбора страницы**
1. Не по макету
2. Кнопка должна быть полностью интерактивной, а не только текст на ней
3. Отступ справа и снизу должен быть одинаковым

![image](https://user-images.githubusercontent.com/12061580/101533413-9d8d1780-39b7-11eb-8c7c-cc358040434e.png)

warmbl committed:
Удалила логотип

warmbl committed:
Новый логотип

warmbl committed:
Выделение полей для некорректных логина и пароля

warmbl committed:
Отредактировала страницу входа в ЕТИС.

IrinaAkimenkova commented:
> # Страница входа
> 1. svg логотипа с прозрачными линиями, поэтому видно, что они друг на друга накладываются. Нужно сделать его полностью непрозрачным, прозрачность задавать через стили.
> 2. Курсор в полях ввода черный.
> 3. Неправильные отступы у текста в полях ввода.
> 4. Шрифт текста в полях ввода отличается от остального.
> 5. Ссылка для регистрации должна быть вертикально выровнена по середине кнопки.
> 6. Информационный текст должен быть прикреплён к низу страницы.
> 7. У формы входа свегда должна центрироваться по вертикали страницы с небольшим смещением вверх.
> 8. Нет одной строки с информацией.
> 9. Нет тени у формы входа.
> 10. Видны белые углы у полей ввода при автозаполнении.

warmbl commented:
> # Страница входа
_> 1. svg логотипа с прозрачными линиями, поэтому видно, что они друг на друга накладываются. Нужно сделать его полностью непрозрачным, прозрачность задавать через стили._
**Исправила.**
_> 2. Курсор в полях ввода черный._
**Исправила**
_> 3. Неправильные отступы у текста в полях ввода._
**Исправила.**
_> 4. Шрифт текста в полях ввода отличается от остального._
**Так надо.**
_> 5. Ссылка для регистрации должна быть вертикально выровнена по середине кнопки._
**Нельзя так сделать. Но чуть-чуть выровняла.**
_> 6. Информационный текст должен быть прикреплён к низу страницы._
**Исправила.**
_> 7. У формы входа свегда должна центрироваться по вертикали страницы с небольшим смещением вверх._
**Есть закомменченый вариант. Но его не надо использовать, потому что слетает ссылка Регистрация (для работодателей).**
_> 8. Нет одной строки с информацией._
**Так надо.**
_> 9. Нет тени у формы входа._
**Теперь есть.**
_> 10. Видны белые углы у полей ввода при автозаполнении._
**Исправила.**

Commit:
Переделал карточки преподавателей на странице "Преподаватели":
- изменил внешние и внутренние отступы и цвет текста;
- убрал рамку вокруг фотографий преподавателей;
- изменил фотографии согласно совету Андрея.

Убрал со страницы "Преподаватели" скриптом лишние <br>.

Пофиксил баг в скрипте для страницы входа (опечатку в if).

ENAleksey commented:
@IrinaAkimenkova

Страница входа
==
### 1. Высота плашки с ошибкой не должна зависеть от высоты страницы
![image](https://user-images.githubusercontent.com/12061580/101776733-37b5a280-3b13-11eb-8d98-8ca3c4d9df35.png)
### 2. Ширина информационного сообщения внизу страницы не должна задаваться в процентах
### 3. Неправильно сделано центрирование информационного сообщения (нужно делать не через отступ слева)
### 4. Текст информационного сообщения не должна заезжаеть на форму входа
![image](https://user-images.githubusercontent.com/12061580/101776612-063cd700-3b13-11eb-9a49-19526199091e.png)
### 5. Сслыка регистрации всё ещё не выровнена относительно кнопки входа
### 6. Из-за добавления border к полям ввода, все элементы смещаются
<div style="display: flex;  justify-content: space-around;">
<img src="https://user-images.githubusercontent.com/12061580/101777117-d3dfa980-3b13-11eb-9e9d-3833c384ce10.png"/>
<img src="https://user-images.githubusercontent.com/12061580/101777148-df32d500-3b13-11eb-9743-e08a91aa05d6.png"/>
</div>

### 7. Шрифт всех input должен наследоваться от родительского элемента
### 8. Внутренний отступ должен быть у формы входа, а не у элементов внутри формы (из-за этого поля ввода не по середине)
<img src="https://user-images.githubusercontent.com/12061580/101812491-3dc07900-3b3d-11eb-8769-16ecac9000d9.png"/>

### 9. Цвет ссылки регистрации темнее чем, нужно из-за opacity: 0.6

ENAleksey commented:
@VladislavMaksimov 

Уведомление в боковом меню всё ещё наезжает на текст

![image](https://user-images.githubusercontent.com/12061580/101779077-9a5c6d80-3b16-11eb-8bf9-d33969037285.png)

Commit:
Добавил в проект иконку расписания.

Commit:
Увеличил отступ между карточками преподавателей.

Подменил иконку расписания. Центрировал её с соответствующим ей текстом (именем препода или названием кафедры).

Commit:
Привёл фотографии преподов к виду как на макете.

Commit:
Поправил отступ сверху блока информации о преподе.

Commit:
Добавил переменную для семейства шрифта и исправил баг с таблицей оценок. Теперь она растягивается на всю ширину.

Commit:
Выровнял иконку расписания с текстом по центру.

Commit:
Исправил баг с бейджем в боковом меню: теперь он не налезает на текст.

VladislavMaksimov commented:
> @VladislavMaksimov
> 
> Уведомление в боковом меню всё ещё наезжает на текст
> 
> ![image](https://user-images.githubusercontent.com/12061580/101779077-9a5c6d80-3b16-11eb-8bf9-d33969037285.png)

Исправил.

Commit:
Добавил в проект иконку заметки.

Commit:
Верстал страницу расписания.

Сделал у блоков с расписанием скругления углов, цвет фона и вннутренние отступы по макету.

Подменил иконки журнала старосты иконками заметок.

VladislavMaksimov commented:
@ENAleksey @MatushkinaAnastasia 

Для страницы сообщений от преподов пока сделал так.

Если стайлить li.week, это задевает страницу расписания. Марджины у всего ul и его обёртки стайлятся криво - сейчас разбираться нет времени. Если марджин менять у первого сообщения с помощью css, то это задевает страницу с объявлениями.

![image](https://user-images.githubusercontent.com/46715602/101892892-c84cbb00-3bc5-11eb-84e7-06e4af77b433.png)

Commit:
Поменял отступ между сообщениями преподов и номерами страниц в "Сообщениях от преподавателей", чтобы не задевать стилями страницы расписания и объявлений.

Commit:
Поменял цвета номерам недель в соответствии с макетом. Поменял скрипты, чтобы они работали на любой вкладке страниц с расписанием и сообщениями от преподов.

Commit:
Пофиксил баг со скриптом для страницы входа. Теперь он работает вне зависимости от того, на какой странице пользователь разлогинился.

Commit:
Добавил в проект иконку зума.

Commit:
Сделал таблицы по макету. Не делал в них то, для чего нужны скрипты.

VladislavMaksimov commented:
@ENAleksey у нас какие-то ховеры задаются для этих элементов на странице расписания?

![image](https://user-images.githubusercontent.com/46715602/101910759-381c6f00-3be1-11eb-8c26-c78746e80d7e.png)

ENAleksey commented:
@VladislavMaksimov 

Страница с расписанием
==
### 1. Цвет аудитории не везде одинаковый
### 2. Не у всех дистанционных пар есть иконки

![image](https://user-images.githubusercontent.com/12061580/101915968-21c5e180-3be8-11eb-80ed-4370363a06d4.png)
### 3. Текст не выравнивается относительно иконки
### 4. Не срабатывает ховер у ссылок на преподавателей
### 5. Не застайлены ссылки и чекбоксы

![image](https://user-images.githubusercontent.com/12061580/101914951-c9daab00-3be6-11eb-93cd-f3ac428a56bb.png)
### 6. Кнопки переключения недели не по макету. Интерактивен только текст.

![image](https://user-images.githubusercontent.com/12061580/101914990-d7903080-3be6-11eb-88ac-6c759d611f91.png)
### 7. Маленький отступ между пустыми парами

![image](https://user-images.githubusercontent.com/12061580/101916292-8ed97700-3be8-11eb-8ded-900027573b5a.png)
### 8. Застайлить плашку с каникулами

![image](https://user-images.githubusercontent.com/12061580/101916374-aa448200-3be8-11eb-9319-a0d87b9ad694.png)
### 9. У всех страниц нет отступа снизу

![image](https://user-images.githubusercontent.com/12061580/101916688-1a530800-3be9-11eb-9ba4-cf9d5f04b63f.png)
### 10. Ссылка на преподавателя не выровнена по середине

![image](https://user-images.githubusercontent.com/12061580/101923585-686c0980-3bf1-11eb-8764-b92d6c064901.png)
### 11. Ссылки на предмет и аудиторию перекрывают ссылку на преподавателя

![image](https://user-images.githubusercontent.com/12061580/101923671-89345f00-3bf1-11eb-952f-4a1ee99f369e.png)
![image](https://user-images.githubusercontent.com/12061580/101923719-97827b00-3bf1-11eb-9cb3-2badcf4671f6.png)

ENAleksey commented:
> @ENAleksey у нас какие-то ховеры задаются для этих элементов на странице расписания?
> 
> ![image](https://user-images.githubusercontent.com/46715602/101910759-381c6f00-3be1-11eb-8c26-c78746e80d7e.png)

Можно менять цвет на чисто белый.

MatushkinaAnastasia committed:
Send Icon

Send Icon for messages

MatushkinaAnastasia committed:
Updated send icon

MatushkinaAnastasia committed:
Delete Send Icon.svg

MatushkinaAnastasia committed:
send icon only for testing

MatushkinaAnastasia committed:
Delete Vector.png

MatushkinaAnastasia committed:
Add files via upload

MatushkinaAnastasia committed:
Delete Vector-test.png

MatushkinaAnastasia committed:
Add files via upload

MatushkinaAnastasia committed:
Delete Send.svg

MatushkinaAnastasia committed:
Add files via upload

MatushkinaAnastasia committed:
Delete Send.svg

MatushkinaAnastasia committed:
Add files via upload

MatushkinaAnastasia committed:
Удалены отступы перед именем преподавателя

MatushkinaAnastasia committed:
Обновление раздела "Сообщения от преподавателей"

Изменения:
1. Изменен цвет карточки нового объявления
2. Убран отступ перед именем (в скрипте)
3. Сделана кнопка "Отправить" 
4. Добавлена иконка в кнопку "Отправить"
5. Сделана форма отправки сообщения преподавателю
6. Немножко изменена форма отправленного сообщения преподавателю

warmbl committed:
Выделение активных элементов для меню и недель.

Commit:
СТАБИЛЬНАЯ ВЕРСИЯ.

Пофиксил плашку у активных элементов бокового меню. Подкорректировал надписи вверху на странице с расписанием.

ENAleksey reviewed (approved):
![image](https://user-images.githubusercontent.com/12061580/101947414-4636b380-3c12-11eb-8ba4-44368a162813.png)

MatushkinaAnastasia committed:
Официальный логотип версии

MatushkinaAnastasia committed:
Delete logo.png

MatushkinaAnastasia committed:
Update manifest.json

MatushkinaAnastasia committed:
Delete official logo.svg

MatushkinaAnastasia committed:
Add files via upload

MatushkinaAnastasia committed:
Update manifest.json

MatushkinaAnastasia committed:
Add files via upload

MatushkinaAnastasia committed:
Update manifest.json

VladislavMaksimov committed:
Merge branch 'master' into dev
